### DISCUSSION TITLE
Can we get some comments for what the template guids go to?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#356] [Q&A]
https://github.com/Dotnet-Boxed/Templates/blob/1f02586c7561891c64b6ed2b50fe581b303a13ca/Source/Content/ApiTemplate/.template.config/template.json#L17

**Comment by RehanSaeed** [comment]
You know what, I can't remember. I searched and couldn't see any reference to them. Perhaps we can even remove them.

**Comment by RehanSaeed** [comment]
Let me know if you find out.

**Comment by VictorioBerra** [comment]
Im 90% sure its unused:

![image](https://user-images.githubusercontent.com/2934507/61884976-b4b28e80-aec2-11e9-9e8e-cb3f8ebeb847.png)

The `113f2d04` one is, but rg ([RipGrep](https://github.com/BurntSushi/ripgrep)) didn't find anything matching the `837bc53e`.

**Comment by RehanSaeed** [comment]
`113f2d04-69f0-40c3-8797-ba3f356dd812` is used for user secrets. Added a comment for this. I think `837bc53e-0271-4e9c-b5b5-c60ea7a7c7b5` is unused and has now been removed.
